x-pack/plugins/observability_solution/entity_manager/server/lib/entities/start_transform.ts

```diff
@@ -26,6 +28,17 @@ export async function startTransform(
         esClient.transform.startTransform({ transform_id: historyTransformId }, { ignore: [409] }),
       { logger }
     );
+    if (isBackfillEnabled(definition)) {
+      const historyBackfillTransformId = generateHistoryBackfillTransformId(definition);
+      await retryTransientEsErrors(
+        () =>
+          esClient.transform.startTransform(
+            { transform_id: historyBackfillTransformId },
+            { ignore: [409] }
```

Comment:
what a 409 means for starting the transform ? that the operation is already in progress ?

Reply:
Correct